src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
What is this import for? Is there an import that can't be unresolved?

Reply:
Yeah, as I added the `jsonconfig.js` file we can now import things from a defined relative path, but the eslint still marks this as unresolved path so thats why I do this disable.

Reply:
I see.. ok. Does that mean we have to have this `eslint-disable` in most other files too?